tests/automation_framework/src/work_order_submit/work_order_submit_params.py

```diff
@@ -595,8 +598,25 @@ def configure_data(
     def configure_data_sdk(
             self, input_json, worker_obj, pre_test_response):
 
+        if "workloadId" not in input_json["params"].keys():
```

Comment:
if I am not wrong this code is for mandatory field remove test, but this is not a good implementation. We are unnecessarily duplicating code. let's discuss on this.